apps/cowswap-frontend/src/common/pure/OrderSubmittedContent/index.tsx

```diff
@@ -51,7 +53,11 @@ export function OrderSubmittedContent({ chainId, account, isSafeWallet, hash, on
     <Wrapper>
       <AnimatedConfirmation />
       <Caption>
-        <Trans>Order Submitted</Trans>
+        {isSafeWallet && safeWallet && safeWallet.threshold > 1 ? (
```

Comment:
I know you are not adding `isSafeWallet` here, but it's now redundant.
I'm in favor of removing it so only `safeWallet` is present.